src/components/page-header/page-header.component.ts

```diff
@@ -152,4 +152,8 @@ export class PageHeaderComponent {
     select(item: PageHeaderNavigation): void {
         this._pageHeaderService.select(item);
     }
+
+    get hasLogoClick(): boolean {
```

Comment:
Just a small thing, generally we put all variables including getters/setters above the constructor, this way every component is consistent and it makes everything very easy to find. 

We also generally prefix private variables with an `_`, I realise this is not technically private and can't be because it needs to be public to access it in the template, however when we have a public property that should only be used in the template it is often good to also prefix it with an underscore, this way anyone consuming this component should know this is an internal property and it should not be used.

Reply:
Moved to above the constructor an added in <code>_</code> prefix